src/site/content/en/blog/granular-chunking-nextjs/index.md

```diff
@@ -0,0 +1,273 @@
+      },
+    },
+  },
+```
+Although including framework-dependent code into a shared chunk means that it can be downloaded and
+cached for any entrypoint, the usage-based heuristic of including common modules used in more than
+_half of pages_ isn't very effective. Modifying this ratio would only result in one of two outcomes:
+more unnecessary code gets downloaded or more code gets duplicated across multiple routes.
+
+To solve this problem, we adopted a [different
+configuration](https://github.com/zeit/next.js/pull/7696) for`SplitChunksPlugin` that reduces
+unecessary code for any route.
+
++   Any sufficiently large third-party module (> 160 KB) is split into its own individual chunk
++   A separate `frameworks` chunk is created for framework dependencies (`react`, `react-dom`,
+    etc…)
++   As many shared chunks as needed are created (up to 25)
+
+This granular chunking strategy provides the following benefits:
+
+1.  **Page load times are improved**. Emitting multiple shared chunks, instead of a single one,
+    minimizes the amount of unneeded (or duplicate) code for any entrypoint.
+2.  **Improved caching during navigations**. Splitting large libraries and framework dependencies
+    into separate chunks reduces the possibility of cache invalidation since both are unlikely to
+    change until an upgrade is made.
+
+## More HTTP requests
+
+`SplitChunksPlugin` defined the basis for granular chunking, and applying this approach to a
+framework like Next.js was not an entirely new concept. Many frameworks, however, still continued to
+use a single heuristic and "commons" bundle strategy for a few reasons. This includes the concern that
+many more HTTP requests can negatively affect site performance.
+
+Browsers can only open a limited number of TCP connections to a single origin (6 for Chrome), so
+minimizing the number of chunks outputted by a bundler can ensure that the total number of requests
+stays under this threshold. However, this only holds true for HTTP/1.1. Multiplexing in HTTP/2
+allows for multiple requests to be streamed in parallel using a single connection over a single
+origin. In other words, we generally do not need to worry about limiting the number of chunks
+emitted by our bundler.
+
+[All major browsers](https://caniuse.com/#feat=http2) support HTTP/2, and we wanted to see for
+ourselves if increasing the number of requests by splitting Next.js's single "commons" bundle into
+multiple shared chunks would affect loading performance in any way. We began by measuring the
+performance of a single site while modifying the maximum number of parallel requests using the
+[`maxInitialRequests`](https://webpack.js.org/plugins/split-chunks-plugin/#splitchunksmaxinitialrequests) property.
+
+![Page load performance with increased number of requests](./performance-num-requests-1.png)
+
+In an average of three runs of multiple trials on a single web page, the load, start-render and
```

Comment:
```suggestion
In an average of three runs of multiple trials on a single web page, the [`load`](https://developer.mozilla.org/docs/Web/API/Window/load_event), Start Render and
```

I'm assuming that `load` is the browser event. Is Start Render a custom metric? You should clarify this.